hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/nio/NioAsyncSocket.java

```diff
@@ -268,13 +269,13 @@ public void flush() {
         }
     }
 
+    @SuppressWarnings("java:S3398")
     private void resetFlushed() {
         flushThread.set(null);
 
-        if (!unflushedBufs.isEmpty()) {
```

Comment:
Can you restore this. I'm a simple person and like simple code.

Reply:
https://github.com/hazelcast/hazelcast/pull/23740/commits/2daff84fb3005813adebe904f51db834144725f7